far2l/src/plug/plugapi.cpp

```diff
@@ -1003,7 +1003,11 @@ static HANDLE FarDialogInitSynched(INT_PTR PluginNumber, int X1, int Y1, int X2,
 		return hDlg;
 
 	// ФИЧА! нельзя указывать отрицательные X2 и Y2
-	if (X2 < 0 || Y2 < 0)
+	const auto fixCoord = [](int first, int second) { return (first < 0 && second == 0) ? 1 : second; };
```

Comment:
is this lambda really needed? I think just using same expression in place will be more readable